packages/jest-worker/src/createWorkerFarm.ts

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import {isAbsolute} from 'path';
+import WorkerFarm from './WorkerFarm';
+import type {WorkerFarmOptions, WorkerModule} from './types';
+
+export type JestWorkerFarm<T> = WorkerFarm & WorkerModule<T>;
+
+const reservedKeys = ['end', 'getStderr', 'getStdout', 'setup', 'teardown'];
```

Comment:
Currently `setup()` and `teardown()` are not excluded. Felt like these should not be present in the returned API.